### DISCUSSION TITLE
Some questions about IP tunnel function

### DISCUSSION THREAD

**Discussion by OptimusTom426** [svpcom/wfb-ng#352] [Questions] [answered]
Here's the message in README : 

 - IP-over-WFB tunnel support. You can transmit ordinary ip packets over WFB link. Note, don't use ip tunnel for high-bandwidth transfers like video or mavlink. It uses less efficient FEC coding and doesn't aggregate small packets.

Is that mean that I can use the full TCP/UDP functionality and freely assign IP addresses to the drone-wfb/gs-wfb virtual network card? In other words, can I use this interface for network communication like in Managed mode?

**Comment by svpcom** [comment, marked as answer]
Yes

**Reply by lida2003** [reply]
Here is a question about how to use IP tunnel over wfb-ng.

I noticed that there are Rx/Tx tunnel box in `wfb-cli gs`. But I didn't find how to assign ip address, and how to use this tunnel?

Is there any document or link avaliable to follow?

**Reply by svpcom** [reply]
See master.cfg. You can override address assignment in /etc/wifibroadcast.cfg
https://github.com/svpcom/wfb-ng/blob/4ea700606c259960ea169bad1f55fde77850013d/wfb_ng/conf/master.cfg#L257
